package.json

```diff
@@ -50,7 +38,19 @@
       "last 1 safari version"
     ]
   },
+  "dependencies": {
+    "@emmetio/codemirror-plugin": "^1.2.3",
+    "codemirror": "^5.60.0",
+    "node-sass": "^6.0.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "4.0.3",
+    "web-vitals": "^1.1.2"
+  },
```

Comment:
moved `dependencies` and `devDependencies` to the bottom of `package.json` file, so we could at least, clarify the package structure of our application.